packages/soci-snapshotter/soci-snapshotter.spec

```diff
@@ -85,17 +89,26 @@ install -p -m 0755 out/soci-snapshotter-grpc %{buildroot}%{_cross_bindir}
 install -p -m 0755 out/soci %{buildroot}%{_cross_bindir}
 install -p -m 0755 out/fips/soci-snapshotter-grpc %{buildroot}%{_cross_fips_bindir}
 install -p -m 0755 out/fips/soci %{buildroot}%{_cross_fips_bindir}
+install -D -p -m 0644 %{S:100} %{buildroot}%{_cross_unitdir}
```

Comment:
You can escape the name of the file here and install `S:100` with the escaped name